### ISSUE TITLE
Can NOT compile using ndk r19 for Android platform

### ISSUE THREAD

**Issue by zack-not-know-why-it-work** [DanielChappuis/reactphysics3d#286]
Hi Daniel

There is a compile error as below when I try to build branch [refactor_shapes_creation](https://github.com/DanielChappuis/reactphysics3d/tree/refactor_shapes_creation) using ndk r19 for Android platform.

Compiler:  ndk r19 

```
/react_physics/include/reactphysics3d/memory/DefaultAllocator.h:67:24: error: use of undeclared identifier
      'aligned_alloc'
                return std::aligned_alloc(GLOBAL_ALIGNMENT, size);
```

**Comment by DanielChappuis** [comment]
Thanks a lot for your feedback. 

I wil try to fix that on Android platform. However, you should really not use this branch. This is a development branch that I am currently working on. You you use the master branch instead.

**Comment by zq-yang** [comment]
> 
Thank you. I will use master branch instead.

**Comment by Chaosaber** [comment]
Hi ， Has this problem been fixed?

**Comment by DanielChappuis** [comment]
This should now be fixed in version [v0.10.0](https://github.com/DanielChappuis/reactphysics3d/releases/tag/v0.10.0) of the library. Thanks a lot for reporting the issue.
